gptme/tools/__init__.py

```diff
@@ -36,84 +27,90 @@
     "ToolSpec",
     "ToolUse",
     "ToolFormat",
+    "Parameter",
+    "ConfirmFunc",
     # functions
-    "execute_msg",
     "get_tool_format",
     "set_tool_format",
-    # files
-    "read_tool",
-    "tool_append",
-    "tool_save",
-    "patch_tool",
-    # code
-    "shell_tool",
-    "python_tool",
-    "gh_tool",
-    # vision and computer use
-    "vision_tool",
-    "screenshot_tool",
-    "computer_tool",
-    # misc
-    "chats_tool",
-    "rag_tool",
-    "subagent_tool",
-    "tmux_tool",
-    "browser_tool",
-    "youtube_tool",
 ]
 
-loaded_tools: list[ToolSpec] = []
+from .save import tool_save
```

Comment:
Unused?

Reply:
Ah no it was this: https://github.com/ErikBjare/gptme/pull/336#discussion_r1894648880

I think we can safely remove it, it's an ancient workaround.

Reply:
Removed.